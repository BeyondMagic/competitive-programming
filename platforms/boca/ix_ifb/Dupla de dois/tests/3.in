4
Godofredo 100
Hortolina 80
Oliveira 10
Joao 0
