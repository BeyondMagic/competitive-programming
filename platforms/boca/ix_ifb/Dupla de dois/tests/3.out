10
Joao 0 Godofredo 100
Oliveira 10 Hortolina 80
